Template/Tag/MatomoTag.web.js

```diff
@@ -126,6 +126,10 @@
                         tracker.disableBrowserFeatureDetection();
                     }
 
+                    if (matomoConfig.disableCampaignParameters) {
```

Comment:
I believe that this should match the name of the config created in `Template/Variable/MatomoConfigurationVariable.php`
```suggestion
                    if (matomoConfig.requireConsentForCampaignTracking) {
```

Reply:
Ok, thanks @snake14!

I wasn't sure as some of the other calls around this were referencing the methods and some were referencing the configs and I couldn't really find a pattern to follow. Will change this and see what the tests do. QA will test the overall change as well.